mitodl/micromasters
# Program Topics

#### What are the relevant tickets?
- Closes #4412

#### What's this PR do?
 - creates a Topic model 
 - add topics (many-to-many) to Program model
 - add topics to Django admin
 - add topics to program API and catalog API

#### How should this be manually tested?
- Using Django Admin, create some new topics and add them to a program or two
- Verify that they appear in `../api/v0/programs` and `../api/v0/catalog` in a format like this:
   ```
       [
            {
                "name": "Computer Science"
            },
            {
                "name": "Engineering"
            },
        ]
```
